stock_restrict_lot_update/tests/test_restrict_lot_update.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2023 Ooops404
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import SavepointCase
+
+
+class TestRestrictLotUpdate(SavepointCase):
```

Comment:
Could you add docstring to each test case?

Reply:
Done!